### ISSUE TITLE
python310Packages.liblarch: fails test at master

### ISSUE THREAD

**Issue by bryanasdev000** [NixOS/nixpkgs#183609]
### Steps To Reproduce
1. Try to build *liblarch*
2. Fails at tests

### Build log
```
...
running install tests
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
running egg_info
writing liblarch.egg-info/PKG-INFO
writing dependency_links to liblarch.egg-info/dependency_links.txt
writing top-level names to liblarch.egg-info/top_level.txt
reading manifest file 'liblarch.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'README'
adding license file 'LICENSE'
adding license file 'AUTHORS'
writing manifest file 'liblarch.egg-info/SOURCES.txt'
running build_ext
Traceback (most recent call last):
  File "/build/source/nix_run_setup", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 35, in <module>
    setup(
  File "/nix/store/w8kdsv5prmkklpqkp8pyhazaknzahfli-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 177, in setup
    return run_commands(dist)
  File "/nix/store/w8kdsv5prmkklpqkp8pyhazaknzahfli-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 193, in run_commands
    dist.run_commands()
  File "/nix/store/w8kdsv5prmkklpqkp8pyhazaknzahfli-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 968, in run_commands
    self.run_command(cmd)
  File "/nix/store/w8kdsv5prmkklpqkp8pyhazaknzahfli-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/setuptools/dist.py", line 1217, in run_command
    super().run_command(command)
  File "/nix/store/w8kdsv5prmkklpqkp8pyhazaknzahfli-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 987, in run_command
    cmd_obj.run()
  File "/nix/store/w8kdsv5prmkklpqkp8pyhazaknzahfli-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/setuptools/command/test.py", line 224, in run
    self.run_tests()
  File "/nix/store/w8kdsv5prmkklpqkp8pyhazaknzahfli-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/setuptools/command/test.py", line 227, in run_tests
    test = unittest.main(
  File "/nix/store/9wa02q541sxq7372f8zv27rl57aribxj-python3-3.10.5/lib/python3.10/unittest/main.py", line 100, in __init__
    self.parseArgs(argv)
  File "/nix/store/9wa02q541sxq7372f8zv27rl57aribxj-python3-3.10.5/lib/python3.10/unittest/main.py", line 124, in parseArgs
    self._do_discovery(argv[2:])
  File "/nix/store/9wa02q541sxq7372f8zv27rl57aribxj-python3-3.10.5/lib/python3.10/unittest/main.py", line 244, in _do_discovery
    self.createTests(from_discovery=True, Loader=Loader)
  File "/nix/store/9wa02q541sxq7372f8zv27rl57aribxj-python3-3.10.5/lib/python3.10/unittest/main.py", line 154, in createTests
    self.test = loader.discover(self.start, self.pattern, self.top)
  File "/nix/store/9wa02q541sxq7372f8zv27rl57aribxj-python3-3.10.5/lib/python3.10/unittest/loader.py", line 349, in discover
    tests = list(self._find_tests(start_dir, pattern))
  File "/nix/store/9wa02q541sxq7372f8zv27rl57aribxj-python3-3.10.5/lib/python3.10/unittest/loader.py", line 405, in _find_tests
    tests, should_recurse = self._find_test_path(
  File "/nix/store/9wa02q541sxq7372f8zv27rl57aribxj-python3-3.10.5/lib/python3.10/unittest/loader.py", line 483, in _find_test_path
    tests = self.loadTestsFromModule(package, pattern=pattern)
  File "/nix/store/w8kdsv5prmkklpqkp8pyhazaknzahfli-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/setuptools/command/test.py", line 57, in loadTestsFromModule
    tests.append(self.loadTestsFromName(submodule))
  File "/nix/store/9wa02q541sxq7372f8zv27rl57aribxj-python3-3.10.5/lib/python3.10/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/build/source/tests/test_liblarch.py", line 23, in <module>
    gi.require_version('Gtk', '3.0')  # noqa
  File "/nix/store/d0m1mk1zbsjysrm1bq44087njrygnl7w-python3.10-pygobject-3.42.1/lib/python3.10/site-packages/gi/__init__.py", line 126, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace Gtk not available
error: builder for '/nix/store/0qsbrpprpa
```

### Additional context

Relevant to https://github.com/NixOS/nixpkgs/pull/181819

### Notify maintainers

@oyren

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.18.10, NixOS, 22.11 (Raccoon), 22.11.20220714.4a01ca3`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.9.1`
 - channels(root): `""`
 - channels(user): `""`
 - nixpkgs: `/etc/nix/channels/nixpkgs`
```

**Comment by bryanasdev000** [comment]
This fail is somewhere in the last 12 days.

**Comment by vcunat** [comment]
https://github.com/NixOS/nixpkgs/pull/181620

**Comment by bryanasdev000** [comment]
> Surely since PR #181620 merged in [29316ab](https://github.com/NixOS/nixpkgs/commit/29316ab92b50fab08382da77b91253e9023c6d11).

Not sure what have broke, but adding buildInputs with gtk3 seems to fix it.

https://github.com/NixOS/nixpkgs/pull/184046
